kernel/aster-nix/src/net/socket/ip/stream/connected.rs

```diff
@@ -67,10 +68,11 @@ impl ConnectedStream {
         }
     }
 
-    pub fn try_sendto(&self, buf: &[u8], flags: SendRecvFlags) -> Result<usize> {
+    pub fn try_send(&self, packet: &[u8], flags: SendRecvFlags) -> Result<usize> {
```

Comment:
My sense is that universally converting the name of the buffer argument in `{try_,}send` to `packet` looks a bit odd, because:
 - We are not necessarily sending a packet. In TCP stream, the user data of multiple system calls can be combined into one packet, and the user data of one system call can be split into multiple packets.
 - So the concept of "packet" does not exist at this point. It will only exist when smoltcp creates outgoing packets from the buffered data.

It _might_ be appropriate to call this argument "packet" in UDP sockets, if emphasizing that they are packets outweighs consistency with TCP sockets. But changing only `send` and skipping `recv` creates another sense of oddity, IMO.